packages/moonstone/Panels/ApplicationCloseButton.js

```diff
@@ -49,11 +58,15 @@ const ApplicationCloseButton = kind({
 		className: 'applicationCloseButton'
 	},
 
-	render: ({backgroundOpacity, onApplicationClose, ...rest}) => {
+	computed: {
+		closeButtonAriaLabel: ({'aria-label': ariaLabel}) => (ariaLabel == null) ? $L('Exit app') : ariaLabel
+	},
+
+	render: ({backgroundOpacity, closeButtonAriaLabel, onApplicationClose, ...rest}) => {
```

Comment:
@ryanjduffy one question. Shall I add `delete rest['aria-label']` here or will be OK since override below?

Reply:
Normally, if we're overriding a prop, we use that prop as the computed value to avoid the confusion. This does work fine as is but I'll update for consistency.